packages/valory/skills/abstract_round_abci/tests/test_tools/test_common.py

```diff
@@ -147,3 +170,66 @@ class TestBaseRandomnessBehaviourTestRunning(BaseRandomnessBehaviourTest):
     randomness_behaviour_class = DummyRandomnessBehaviour  # type: ignore
     next_behaviour_class = DummyKeeperSelectionBehaviour  # type: ignore
     done_event = Event.DONE
+
+
+class TestBaseSelectKeeperBehaviourTestSetup(BaseCommonBaseCaseTestSetup):
+    """Test BaseRandomnessBehaviourTest setup."""
+
+    _test_cls = BaseSelectKeeperBehaviourTest
```

Comment:
I refactored a bit to reduce repetition, but still highly repetitive, in part due to variations such as `randomness_behaviour_class` vs `select_keeper_behaviour_class` attributes, which could be generalized to `behaviour_class`